examples/percentage_of_actors.rb

```diff
@@ -19,7 +19,7 @@ def initialize(id)
   alias_method :flipper_id, :id
 end
 
-total = 10_000
+total = 100_000
```

Comment:
awesome, will probably want to update [percentage_of_time example](https://github.com/jnunemaker/flipper/blob/master/examples/percentage_of_time.rb) too